### PR TITLE
[New] Open mic announcement for 2024-10-04

### PR DESCRIPTION
**Pull request by blcham**

### CODE REVIEW

**Comment by blcham** [comment]
@MichalMed Since it took me that long, I am merging PR. Please revise and i will make separate PR to fix.

**Comment by ledsoft** [comment]
Your post is missing the first sentence that is used on the main page, see https://kbss.felk.cvut.cz/web/

File: _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "AI tools to power academic research"
+categories: [Open Mic Announcement, Open Mic]
+---
+
```
Comment (blcham) [inline _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown:5]: I left this empty space and thus no description was shown

File: _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: "AI tools to power academic research"
```
[the remark this answers is not in the document]
Comment (blcham) [reply _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown:2]: fixed

File: _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+categories: [Open Mic Announcement, Open Mic]
+---
+
+<!-- Naming convention: post MUST be named beginning with YEAR-MM-DD-title.MARKDOWN. Create a COPY of this file in _posts and rename it according to the convention. The date is not date of presentation, but date of publication of the announcement. The title in human readable form is put in the quotes to the title parameter in front matter  -->
+
+Content: The Open mic session starts on Friday 4 October 2024 at 10:30 via [this link](https://meet.jit.si/open-mic-kbss). The speaker is [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško) who is going to talk about AI tools to help improve academic research.
```
[the remark this answers is not in the document]
Comment (blcham) [reply _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown:8]: fixed

File: _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown (hunk 1 above)
Comment (blcham) [reply _posts/2024-10-02-ai-tools-to-power-academic-research-announcement.markdown:5]: fixed

**Pull request merged**